lib/common_test/src/ct_snmp.erl

```diff
@@ -273,10 +282,13 @@ start_application(App) ->
     end.
  
 -doc """
-stop(Config) -> ok
+stop(Config) -> ok | {'EXIT', Term}
```

Comment:
I think this should be fixed in code, it's obvious that intentions were to always return ok.

Reply:
Fixed in code.